app/routes.py

```diff
@@ -1 +1,306 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+import requests
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix=("/tasks"))
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# create route (get) to get all tasks
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+    tasks_response = []
+    sort = request.args.get("sort")
+    
+    if sort == "asc":
+        tasks = Task.query.order_by(Task.title.asc()).all()
+    elif sort == "desc":
+        tasks = Task.query.order_by(Task.title.desc()).all()
+    else:
+        tasks = Task.query.all()
+        
+    for task in tasks: 
+        tasks_response.append({
+            "id": task.task_id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete": task.completed_at != None
+        })
+        
+    return jsonify(tasks_response)
+
+
+# create a task
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    
+    if "title" not in request_body or "description" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
+    
+    new_task = Task(
+        title = request_body.get("title"),
+        description = request_body.get("description"),
+        completed_at = request_body.get("completed_at")
+        )
+    
+    db.session.add(new_task)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model